File: 1-Iniciante/ex1037.c
#include <stdio.h>

int main ()
{
    float valor;

    scanf("%f", &valor);

    if (valor > 100 || valor < 0)
    {
        printf("Fora de intervalo\n");
    }
    else if (valor > 75 && valor <= 100)
    {
        printf("Intervalo (75,100]\n");
    }
    else if (valor > 50 && valor <= 75)
    {
        printf("Intervalo (50,75]\n");
    }
    else if (valor > 25 && valor <=50)
    {
        printf("Intervalo (25,50]\n");
    }
    else
    {
        printf("Intervalo [0,25]\n");
    }
    
    return 0;
}

File: 1-Iniciante/ex1037.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let numero = parseFloat(input);

// processamento & saida
if(numero < 0.0 || numero > 100.0) console.log("Fora de intervalo");
else if(numero <= 25.0) console.log('Intervalo [0,25]');
else if(numero <= 50.0) console.log('Intervalo (25,50]');
else if(numero <= 75.0) console.log('Intervalo (50, 75]');
else console.log("Intervalo (75,100]");